Commands_Functions/create_command.py

```diff
@@ -0,0 +1,33 @@
+import os, json
+from schema_keys import Keys
+from icommand import Icommand
+from output.exceptions import MissingInput, WrongInput
+from output.status import Status
+
+class CreateCommand(Icommand):
+    def __init__(self, schema_path):
+        self.schema_path = schema_path
+        self.isvalid()
+        
+    def isvalid(self):
+        if self.schema_path == None or not os.path.exists(self.schema_path) :
+            raise MissingInput(status = Status.MissingInput, message = "the schema is missing")
+        self.schema_file = self.schema_path
+        with open(self.schema_path, 'r') as schema: 
+            try:
+                self.schema_data = json.load(schema)
+            except:
+                raise WrongInput(status = Status.WrongInput, message = "schema is written in a wrong json format")
+
+    def execute(self):
+        schema_data = self.schema_data
```

Comment:
when I wrote it, I wrote it because I wanted to bring the schema_data variable from the ```init``` to the ```execute``` by assigning it to the variable ```schema_data``` but now after learning more about python class, it is useless as I can say in the ```execute``` function ```self.schema_data```. I solved it.